druid/ingestion-spec/rollups/ml_project_index_rollup.json

```diff
@@ -0,0 +1,258 @@
+            },
+            {
+              "type": "string",
+              "name": "designation"
+            },
+            {
+              "type": "string",
+              "name": "task_evidence"
+            },
+            {
+              "type": "string",
+              "name": "task_id"
+            },
+            {
+              "type": "string",
+              "name": "status_of_project"
+            },
+            {
+              "type": "string",
+              "name": "program_name"
+            },
+            {
+              "type": "string",
+              "name": "createdBy"
+            },
+            {
+              "type": "string",
+              "name": "program_externalId"
+            },
+            {
+              "type": "string",
+              "name": "private_program"
+            },
+            {
+              "type": "string",
+              "name": "project_terms_and_condition"
+            },
+            {
+              "type": "string",
+              "name": "state_externalId"
+            },
+            {
+              "type": "string",
+              "name": "block_externalId"
+            },
+            {
+              "type": "string",
+              "name": "district_externalId"
+            },
+            {
+              "type": "string",
+              "name": "cluster_externalId"
+            }
+          ],
+          "dimensionExclusions": []
+        },
+        "metricsSpec": [
+          {
+            "type": "HLLSketchBuild",
+            "name": "count_of_unique_programs",
+            "fieldName": "program_id"
+          },
+          {
+            "type": "HLLSketchBuild",
+            "name": "count_of_createdBy",
+            "fieldName": "createdBy"
+          },
+          {
+            "type": "HLLSketchBuild",
+            "name": "count_of_solution_id",
+            "fieldName": "solution_id"
+          }
+        ],
+        "granularitySpec": {
+          "type": "uniform",
+          "segmentGranularity": "DAY",
+          "queryGranularity": "HOUR",
+          "rollup": true
+        },
+        "transformSpec": {
+          "filter": null,
+          "transforms": [
+            {
+              "type": "expression",
+              "name": "task_evidence_status",
+              "expression": "if(array_length(\"task_evidence\") > 0, 'true', 'false')"
+            },
+            {
+              "type": "expression",
+              "name": "parent_channel",
+              "expression": "upper('SHIKSHALOKAM')"
+            },
+            {
+              "type": "expression",
+              "name": "project_created_type",
+              "expression": "if((NULL != \"project_created_type\"), 'project imported from library', 'user created project')"
+            }
+          ]
+        }
+      },
+      "ioConfig": {
+        "topic": "{{env}}.ml.project",
+        "replicas": 1,
+        "taskCount": "{{rollup_ml_project_taskcount}}",
```

Comment:
this variable is not added in defaults

Reply:
Added now. Please check

This ingestion spec was merged last time by Anand P and didnot make any changes now. So i didnot change